docs/hive-integration/README.md

```diff
@@ -110,23 +110,23 @@ test-infra-cluster-assisted-installer
 test-infra-cluster-assisted-installer-master-2
 SpecSynced	The Spec has been successfully applied
 Connected	The agent's connection to the installation service is unimpaired
-ReadyForInstallation	The agent cannot begin the installation because it has already started
+RequirementsMet	The agent cannot begin the installation because it has already started
```

Comment:
This info should be changed to something more positive like just "Installation already started"
WDYT? @filanov

Reply:
+1

Reply:
Maybe installation in progress?

Reply:
`Requirements met. The agent's installation is in progress.` WDYT?

Reply:
Or, `Installation already started and is in progress`

Reply:
`Installation already started and is in progress` sounds good to me

Reply:
Done